### ISSUE TITLE
Create a detailed roadmap

### ISSUE THREAD

**Issue by Seb-stian** [ObsidianMC/Obsidian#255]
This issue should address the following problems:
- contributors have problem with identifying parts that are missing → it is difficult to assign tasks
- we have poor estimate of how much resources it will take to finish (certain parts of) Obsidian

We should work on a more detailed Roadmap that will list **everything** that needs to be implemented and our progression. It would be valuable to:
- put them in chronological order of when they should be implemented
- track progression
- keep estimates in eg. man-hours

I believe this will be beneficial to occasional contributors and the internal team alike.

We should discuss the form. A single markdown file in this repository may cut it, but it may also add extra noise. A separate repository would keep this one clean and open more oppurtunities, but also add a need of keeping it up to date (which we are historically bad at *cough cough* API documentation *cough cough*).

**Comment by Naamloos** [comment]
Agreed, but I do think that it might be better to not estimate how long a thing would take as that adds extra pressure to finish a task "on time".

**Comment by Seb-stian** [comment]
@Naamloos Good point. It'd be nice to have sense of getting closer to the goal though. Some metric to go by, other than:
- [ ] not done
- [ ] not done
- [X] done
- [X] done

Like for example:
**WORLD GENERATION**
![](https://us-central1-progress-markdown.cloudfunctions.net/progress/50)

**Comment by Naamloos** [comment]
not a bad idea, that's fun

**Comment by ghost** [comment]
So how is it going?

**Comment by Tides** [comment]
Going back to this we can use the milestones feature here, create them and not set a due date. And just add issues to them as they come 

![Image](https://github.com/user-attachments/assets/d934fe84-1d3e-4cb6-ac94-6cf22627c9bf)
